shared/codec/proto/codec_test.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package utility;
+
+option go_package = "github.com/pokt-network/pocket/shared/codec";
+
+message TestProtoStructure {
```

Comment:
I don't fully understand the need/purpose of this test structure, which likely means it needs some additional documentation.

Can you explain what it's for? We can probably move that explanation in comments or a README for future readers too.

Reply:
Do you understand the basis of this issue? 

I can be more explicit in code if that's what you're asking